<launch>
  <arg name="use_sim" default="true" />

  <!-- Launch the pick_place.py node -->
  <node pkg="robot_manipulation" type="pick_place.py" name="pick_place_node" output="screen">
  </node>

  <!-- <node pkg="robot_manipulation" type="box_detector.py" name="box_detector_node" output="screen">
    <remap from="/detections" to="/box_detector/detections"/>
    <remap from="/debug" to="/box_detector/debug"/>
  </node> -->

  <!-- <node pkg="robot_manipulation" type="box_detector2.py" name="box_detector_node" output="screen">
    <remap from="/point_cloud" to="/camera/depth/color/points" />
    <remap from="/debug" to="/box_detector/debug"/>
  </node> -->
  
  <!-- Launch the base functionality -->
  <include file="$(find robot_manipulation)/launch/base.launch">
    <arg name="use_rviz" value="true" />
    <arg name="use_sim" value="$(arg use_sim)" />
    <!-- Unfortunately I can't get it to load custom transforms, so I'm going to publish them manually. That file is not necessary anymore. -->
    <!-- <arg name="transform_filepath" value="$(find robot_manipulation)/config/static_transforms.yaml" /> -->
  </include>
</launch>
